### ISSUE TITLE
The term 'PRIVATE_KEY_PASSWORD=password' is not recognized as a name of a cmdlet, function, script file, or executable program.

### ISSUE THREAD

**Issue by JPLACLAU** [PatrickAlphaC/ethers-simple-storage-fcc#36]
I'm using VScode in windows 10 with admin rights.
Regarding instructions on 7:47 on the video. I cannot define the variable PRIVATE_KEY_PASSWORD when writing
"PRIVATE_KEY_PASSWORD=password node deploy.js"  For some reason I cannot define any variable on the integrated powershell terminal of the VScode, executed with or without admin rights.
This is the error message that I have when writing that command line:

PS E:\mycodefolder> PRIVATE_KEY_PASSWORD=password node deploy.js

Error:
"PRIVATE_KEY_PASSWORD=password: The term 'PRIVATE_KEY_PASSWORD=password' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again."

I would greatly thank in advance any sort of help, I'm sure this is just a newbie problem that I encountered.

**Comment by PatrickAlphaC** [comment]
Are you using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)?

**Comment by JPLACLAU** [comment]
> Are you using [WSL](https://docs.microsoft.com/en-us/windows/wsl/install)?

So I checked if it was installed using: "wsl -l -v". And it was but it was "Stopped".:
  NAME      STATE           VERSION
* Ubuntu    Stopped         2

I turned it on using "wsl npm init" (Why was it stopped? Do I have to turn it on each day after turning on the computer again?)
  NAME      STATE           VERSION
* Ubuntu    Running         2

Now that it is "Running", I decided to try again " PS E:\mycodefolder> PRIVATE_KEY_PASSWORD=password node deploy.js"
But the answer is still the same:
"PRIVATE_KEY_PASSWORD=password: The term 'PRIVATE_KEY_PASSWORD=password' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again."

Any other ideas? I thank you in advance for your time.
